content/libraries/klc_history.adoc

```diff
@@ -5,6 +5,12 @@ url = "/libraries/klc/history/"
 +++
 
 ---
+== 3.0.22 - 2019-09-20
+* Clarified KiCad footprint assignment work-flows as generic vs fully specified symbols (G2.1)
+  * Replace the word atomic with fully specified as this term better fits the used methodology.
```

Comment:
This bullet is the only one ending in a period. Also, can you capture the KLC clauses that were changed for each bullet?

Reply:
done